### DISCUSSION TITLE
Is there a way to not flip the viewrenderable when viewing from behind?

### DISCUSSION THREAD

**Discussion by thupten1234** [sceneview/sceneform-android#377] [Q&A]
Is there a way to not flip the viewrenderable when viewing from behind? When walking around the viewrenderable, the view flips to become readable. Is there a way to disable that?

**Comment by grassydragon** [comment]
Hi!
Currently not, but it is possible to add a parameter to the view material and also check it here: https://github.com/SceneView/sceneform-android/blob/993ad53f31228de3918ead4cd9509ed10347304d/core/assets-sources/materials/sceneform_view_material.mat#L35
You can create a pull request with necessary modifications.
